build/cqs.tsv

```diff
@@ -312,16 +315,17 @@ org.apache.curator:curator-test	2.7.1	test
 org.apache.hadoop:hadoop-minicluster	2.8.5	test
 org.apache.hbase:hbase-testing-util	1.4.12	test
 org.apache.hbase:hbase-testing-util	2.2.3	test
-org.apache.kafka:kafka_2.11	test:2.1.1	test
 org.apache.kafka:kafka-clients	test:2.1.1	test
+org.apache.kafka:kafka_2.11	test:2.1.1	test
 org.cassandraunit:cassandra-unit	2.2.2.1	test
 org.codehaus.groovy:groovy-jsr223	2.4.5	test
-org.geotools:gt-geometry	23.3	test
 org.geotools.jdbc:gt-jdbc-h2	23.3	test
+org.geotools:gt-geometry	23.3	test
```

Comment:
weird, seems like the sort order changed, must be something specific to the local environment. I'll see if I can standardize it in the build script.